examples/kmm-sample/iosApp/iosApp/ContentView.swift

```diff
@@ -4,6 +4,30 @@ import shared
 struct ContentView: View {
     let calculator = Calculator.Companion()
     let greet = Greeting().greeting()
+    @State var token: LibraryCancellable? = nil
+    var counter: Counter
+    
+    class Counter: ObservableObject {
+        var count: Int
+        init(count: Int) {
+            self.count = count
+        }
+    }
+    
+    init () {
+        counter = Counter(count : self.calculator.history().count)
+    }
+    
+    private func listen()-> LibraryCancellable {
+        do {
+            return try calculator.listen {
+                self.counter.count = self.calculator.history().count
+                print("History updated \(self.counter.count)")
+            }
+        } catch {
+            print("Failed to register for notifications: \(error)")
```

Comment:
This isn't particularly useful to the user– but I'm not sure what alternative you would want in this sample code. Considering this is a precondition of the entire sample working, I would probably just `fatalError` with the appropriate message